research/format.markdown

```diff
@@ -0,0 +1,97 @@
+# Research on `format` keyword 
+Changes in `format` Keyword Behaviour Across all JSON Schema Drafts: Additions, Deletions, and Upgrades.
+
+## Draft 0 to Draft 2 (No changes)
```

Comment:
Let's not include content on the titles. Please put "No changes" as a content paragraph of its own